src/app.js

```diff
@@ -0,0 +1,41 @@
+import './app.scss';
+import { createElement } from './lib/dom';
+import { title } from './components/title';
+import { createNewQuoteButton } from './components/button';
+
+export function app() {
+  const header = createElement('header', {
+    className: 'header'
+  });
+  const main = createElement('main', {
+    className: 'main'
+  });
+  const titleElement = title('Simpsons Quotes');
+  const showButton = createNewQuoteButton();
+
+  const quote = fetchData();
```

Comment:
`fetchData()` is async and returns a Promise. 
Thus, `quote` is not the expected data.

You have to wait for the Promise to be fulfilled to receive the data or quote.

I would suggest creating a new function to append a new quote to your `quoteElement`:

```js
async function addQuote() {
  const quote = await fetchData();
  quoteElement.appendChild(document.createTextNode(quote.quote));
}
```

If you call `addQuote()`, it is still async, but you don't have to wait for the result.


An alternative (using `then` instead of `await`):
```js
fetchData().then(quote => {
  quoteElement.appendChild(document.createTextNode(quote.quote));
});
```

Reply:
I want to try to combine the button with an eventlistener and the api function. But I keep that in mind.